### PR TITLE
Adding more weathers for VIP

### PR DESCRIPTION
**Pull request by FreakinsPL**
Thank you for making a contribution to [Las Venturas Playground](https://sa-mp.nl/)!

Please take a moment to fill in the questions in this template, to make it easier for our developers
to understand what you're doing.

## What is being changed?
 - VeryImportantPlayersManger.pwn
 - VeryImportantPlayersCommands.pwn

## Why is this being changed?
 - Update

## How are you making the change?
  - I have a working check-out of Las Venturas Playground.
  - I have included tests for any changed JavaScript code.
  - I have tested this change myself on my local server.

### CODE REVIEW

**Comment by EPIC-striker** [comment]
Hey Freakins, 

I have tested your changes on my local server and they worked fine!
Even though I did not saw VIP players asking for this I think they would appreciate these additional weathers.

File: pawn/Features/Gameplay/VIP/VeryImportantPlayersCommands.pwn [hunk 1]
```diff
@@ -33,7 +33,7 @@ class VeryImportantPlayersCommands {
         }
 
         if (Command->parameterCount(params) != 1) {
-            SendClientMessage(playerId, Color::Information, "Usage: /my weather [day/evening/hot/storm/foggy/drugs]");
+            SendClientMessage(playerId, Color::Information, "Usage: /my weather [day/evening/underwater/sandstorm/drugs/hot/cloudy/storm/foggy]");
```
Comment (OttoRocket-LVP) [inline pawn/Features/Gameplay/VIP/VeryImportantPlayersCommands.pwn]: I suggest to keep the order of the first ones the same.

They'll be the most relevant weathers for the players to use, having those as first reads won't cludge it too much.
You could also possibly go for an alphabetical order.

File: pawn/Features/Gameplay/VIP/VeryImportantPlayersManager.pwn [hunk 2]
```diff
@@ -54,7 +63,7 @@ class VeryImportantPlayersManager {
             SetPlayerWeather(playerId, -66);
             TimeController->setPlayerOverrideTime(playerId, 12, 0);
         } else {
-            SendClientMessage(playerId, Color::Information, "Usage: /my weather [day/evening/hot/storm/foggy/drugs]");
+            SendClientMessage(playerId, Color::Information, "Usage: /my weather [day/evening/underwater/sandstorm/drugs/hot/cloudy/storm/foggy]");
```
Comment (OttoRocket-LVP) [inline pawn/Features/Gameplay/VIP/VeryImportantPlayersManager.pwn]: Same for this, try to keep the order same and add the new ones last or go for an alphabetical order

**Comment by OttoRocket-LVP** [comment]
Thank you for your contribution!

It's looking good in general :)

File: pawn/Features/Gameplay/VIP/VeryImportantPlayersCommands.pwn (hunk 1 above)
Comment (FreakinsPL) [reply pawn/Features/Gameplay/VIP/VeryImportantPlayersCommands.pwn]: Done :)

File: pawn/Features/Gameplay/VIP/VeryImportantPlayersManager.pwn (hunk 2 above)
Comment (FreakinsPL) [reply pawn/Features/Gameplay/VIP/VeryImportantPlayersManager.pwn]: Done :)

**Comment by OttoRocket-LVP** [comment]
Awesome, thank you!

I'll merge it as striker mentioned it's working.

**Pull request merged**